docs/develop/01_sdk/04_How to integrate/04_vitejs.md

```diff
@@ -0,0 +1,77 @@
+---
+id: howto-integrate-vitejs
+title: ViteJS
+---
+
+[ViteJS](https://vitejs.dev/guide/) is a great tool to bootstrap your project.
+As for all browser based projects, you have to make sure that nodejs polyfills are in place to make the SDK work.
+If you start from any ViteJS template, you only have to make sure to install and enable the polyfills.
+
+Bootstrap using the template of your choice and install KILT + NodeJS polyfills:
+
+```
+yarn create vite my-app --template react-ts
```

Comment:
We can do it with the template but I like to keep the instructions because they are valid for any template vitejs provides. So it would be exactly the same if you rather want a vuejs app.

I will finish the template and put it in the kilt org github. I'll create it as private repo for now, then we can give it some polishing (you said it would still need some styling) and then we can add a link to it to this section and make it public.

Are you ok with that dual approach?

Reply:
It's a bit more maintenance effort, so it's ok but not optimal. Do we have to show people how to add polyfills to the `vite.config.ts`, or can we link to docs of vite here?

Reply:
It wasn't too easy to figure it out, having instructions here makes sense imho